packages/nuxt/README.md

```diff
@@ -53,20 +53,24 @@ If the setup through the wizard doesn't work for you, you can also set up the SD
 
 The Sentry Nuxt SDK is based on [Nuxt Modules](https://nuxt.com/docs/api/kit/modules).
 
-1. Add `@sentry/nuxt` to the modules section of `nuxt.config.ts`:
+1. Add `@sentry/nuxt/module` to the modules section of `nuxt.config.ts`:
```

Comment:
The module is included during build time, but can (and will) include client and server runtime plugins. So I think `/module` is still fitting here. Or do you suggest creating two modules? 🤔  I don't think this would make sense because we still have the server/client plugins where we can differentiate between the runtimes.

Reply:
I think `@sentry/nuxt/module` makes sense